kikuchipy/signals/ebsd.py

```diff
@@ -1674,16 +1673,11 @@ def hough_indexing(
         sig_shape = am.signal_shape[::-1]
         step_sizes = tuple([a.scale for a in am.navigation_axes[::-1]])
 
-        # Check indexer
-        phase_list_pei = _get_pyebsdindex_phaselist(phase_list)
+        # Check indexer (but not the reflectors)
         _ = _indexer_is_compatible_with_kikuchipy(
             indexer, sig_shape, nav_size, raise_if_not=True
         )
-        if indexer.phaselist != phase_list_pei:
-            raise ValueError(
-                f"`indexer.phaselist` {indexer.phaselist} and the list determined from"
-                f" `phase_list` {phase_list_pei} must be the same"
-            )
+        _ = _phase_lists_are_compatible(phase_list, indexer, raise_if_not=True)
```

Comment:
If these functions take any decent time you could consider [caching](https://docs.python.org/3/library/functools.html) the results but if they are quick I wouldn't worry about it.

Reply:
An IPython %timeit returned:
* `_indexer_is_compatible_with_kikuchipy()`: 1.61 us +/- 264 ns
* `_phase_lists_are_compatible()`: 54 us +/- 1.51 us

These Hough indexing methods of the EBSD class are not meant to be called hunderds of times in a loop, so I think this is OK.